### ISSUE TITLE
Browse category titles are unreadable because of poor color contrast

### ISSUE THREAD

**Issue by escowles** [pulibrary/pomegranate#492]
examples:
* https://dpul.princeton.edu/ga_treasures/browse
* https://dpul.princeton.edu/islamicmss/browse
* https://dpul.princeton.edu/wa/browse

**Comment by christinach** [comment]
from axe:
`To solve this violation, you need to:
Fix the following:
Element has insufficient color contrast of 4.07 (foreground color: #777777, background color: #f5f4f1, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1`
